rewriter/Prop.cc

```diff
@@ -152,11 +152,32 @@ optional<PropInfo> parseProp(core::MutableContext ctx, const ast::Send *send) {
         }
         case core::Names::merchantProp().rawId():
             ret.isImmutable = true;
+            // This is a lie; technically the API allows an optional `merchant:` keyword arg to
+            // customize the method name. It defaults to `:merchant`
             ret.name = core::Names::merchant();
             // 5 is the length of the _prop suffix
             ret.nameLoc = core::LocOffsets{send->loc.beginPos(), send->loc.endPos() - 5};
             ret.type = ast::MK::Constant(send->loc, core::Symbols::String());
             break;
+        case core::Names::merchantTokenProp().rawId():
+            ret.isImmutable = true;
+            // This is a lie; technically the API allows an optional `merchant:` keyword arg to
```

Comment:
```suggestion
            // This is a lie; technically the API allows an optional `name:` keyword arg to
```

Reply:
Will defaulting the name to `:merchant` be problematic for go-to-definition or the compiler?

Reply:
it's definitely not the right thing to do, but it's the same problem that used to exist for `merchant_prop`, and that case hasn't come up yet, so I'm not worried about fixing it in this PR